packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -457,6 +457,11 @@ const EVENTS = {
     group: GROUPS.qrConnectDevice,
     event: 'fx_view_engage',
   },
+
+  'cad.notnow.engage': {
+    group: GROUPS.connectDevice,
+    event: 'notnow_engage',
```

Comment:
I'm going to change this to `cad_notnow_engage` because there will be a "Not Now" link on /pair (FXA-5842), so I'm going to use prefixes to distinguish them.

Reply:
👍 I came back to re-review but since you might have a couple more changes anyway I'll circle back early tomorrow.